pages/linux/blkdiscard.md

```diff
@@ -0,0 +1,15 @@
+# blkdiscard
+
+> Discards device sectors on storage devices, useful for SSDs. 
```

Comment:
Looks like there's a trailing space that's causing the issue :-)

Also, what about this?

```suggestion
> Discards device sectors on storage devices. Useful for SSDs.
```

Is this like the `fstrim` command?

Reply:
The `fstrim` command is somewhat similar. Both are primarily for SSDs, but `fstrim` discards unused blocks. `blkdiscard` doesn't care if the blocks are used or not.